src/fuzz/fuzz-dhcp-server.c

```diff
@@ -62,7 +62,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         server->bound_leases[pool_offset] = lease;
         assert_se(hashmap_put(server->leases_by_client_id, &lease->client_id, lease) >= 0);
 
-        dhcp_server_handle_message(server, (DHCPMessage*)data, size);
+        assert_se(dhcp_server_handle_message(server, (DHCPMessage*)data, size) >= 0);
```

Comment:
`data` can be [any sequence of bytes](http://llvm.org/docs/LibFuzzer.html#fuzz-target), so it seems to be pretty normal for `dhcp_server_handle_message` to return negative values. It would probably be better to put `(void)` here to make `Coverity` happy.